src/sc/sidechaintypes.h

```diff
@@ -500,6 +507,15 @@ struct ScFeeData_v2 : public ScFeeData {
     {
         return ScFeeData::operator==(rhs) && submissionHeight == rhs.submissionHeight;
     }
+
+    virtual std::string ToString() const
```

Comment:
Shouldn't this be an `override` instead of `virtual` ?

Reply:
Yes makes sense, thanks for spotting!